pcap-config.in

```diff
@@ -45,7 +46,12 @@ do
 		;;
 
 	-h|--help)
-		echo "Usage: pcap-config [ --help ] [ --static | --static-pcap-only ] [ --libs | --additional-libs ]"
+		echo "Usage: pcap-config [ --help ] [--version] [ --static | --static-pcap-only ] [ --libs | --additional-libs ]"
+		exit 0
+		;;
+
+	--version)
+		echo "$VERSION"
```

Comment:
Any reason not to just say `echo "@PACKAGE_VERSION@"` here, and omit the `VERSION` variable?

Reply:
No particular reason, it just felt consistent with having a list of configure-replaced variables up above. And now that I check again it turns out that it's sometimes done that way and sometimes not, so...

Doing a bit of archaeological digging around, commit 16f6b49c87c030be6c4ffc5890a93309f810c849 moved a bunch of things to the top.
Commit 2fda02357aeba9bdd3223e59aa7541fc144db10f and commit c09246855fdce36b463902901d37ffbdd80d3635 moved a couple things back inline.

...

Whatever you prefer is fine.